modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/node/NodeStorage.scala

```diff
@@ -58,7 +58,7 @@ object NodeStorage {
         }
 
       def nodeStates: Stream[F, NodeState] =
-        nodeStateTopic.subscribe(1)
+        nodeStateTopic.subscribe(100)
```

Comment:
Because there were missing events. Not all states have gossiped all the time even though the state change successfully transitioned. This is a temporary workaround, to allow testing. I'm working on a permanent fix for this issue.